### ISSUE TITLE
Carousel Prevents People With Disabilities From Reading Front Page Effectively

### ISSUE THREAD

**Issue by jhomme** [microsoft/vscode-docs#3956]
Hi,
The carousel on the front page of the docs cannot bestopped, started, or paused via keyboard. This prevents screen reader users and people with motor disabilities from reading the page properly.

**Comment by gregvanl** [comment]
Duplicate of https://github.com/microsoft/vscode-docs/issues/3715 fixed July 2020
